src/realm/sync/client.cpp

```diff
@@ -405,7 +405,9 @@ ClientImpl::~ClientImpl()
 void ClientImpl::cancel_reconnect_delay()
 {
     // Thread safety required
-    auto handler = [this] {
+    event_loop_post([this](Status status) mutable {
```

Comment:
what changed here that needed the lambda to be mutable?

Reply:
This was left in by mistake - removed.